README.md

```diff
@@ -71,10 +71,21 @@ then slide `mkFlake` between your outputs function head and body,
 
 Now you can add the remaining module attributes like in the [the template](./template/default/flake.nix).
 
-# Example
+# Templates
 
 See [the template](./template/default/flake.nix).
 
+# Examples
+
+See the folder [examples/](./examples).
+
+# Projects using flake-parts
+
+- [nixd](https://github.com/nix-community/nixd/blob/main/flake.nix) (c++)
+- [hyperswitch](https://github.com/juspay/hyperswitch/blob/main/flake.nix) (rust)
+- [argo-workflows](https://github.com/argoproj/argo-workflows/blob/master/dev/nix/flake.nix) (go)
+- [nlp-service](https://github.com/recap-utr/nlp-service/blob/main/flake.nix) (python)
+
```

Comment:
I'd add hercules-ci-agent, but that one's too complicated for an example.

```suggestion
- [haskell-flake examples](https://haskell.flake.page/examples) (haskell)
```

@srid Do you have a single, particularly nice to read real world example to put here instead?

Reply:
If you want real-world example, then: https://github.com/srid/emanote/blob/master/flake.nix